### DISCUSSION TITLE
HTTP Server

### DISCUSSION THREAD

**Discussion by vanasthomas** [MaJerle/lwesp#73] [Q&A]
Hi 

Thanks for the library, its working nicely, I am trying to run HTTP web server with the library and see there is HTTP sub section in the apps..I have gone through all the examples but cant seem to find how to use the HTTP part. Is there a basic example on how to use it and serve a basic HTML page?

**Comment by MaJerle** [comment]
I have tried to implement asynchronous HTTP using callbacks and state machines and it partially worked, tested on win32.
For embedded, target was to implement SD card reading of the files and then to have possibility to replace some tags in the HTML (SSI tags) with data.

However work has never been 100% completed.

**Reply by vanasthomas** [reply]
Ah ok, thanks for the info, will see what plan I can make
